src/lib/features/listening-history/model/store.ts

```diff
@@ -31,12 +35,23 @@ const addToLocalListeningHistory = (podcastId: number) => {
 };
 
 export const addToListeningHistory = async (episode: Episode) => {
-  const id = episode.podcastID;
-  addToLocalListeningHistory(id);
-  try {
-    addToCloudListeningHistory(id);
-  } catch (e) {
-    alert((e as Error).message);
+  addToLocalListeningHistory(episode.podcastID);
+  if (get(user) !== null && window.navigator.onLine) {
```

Comment:
The nesting in this function is a little extra. How about this gets converted into a guard statement?
```suggestion
  if (get(user) === null || !window.navigator.onLine) return;
```

Reply:
I don't like them. They're not expressive, they read like a machine, not like a human